package-lock.json

```diff
@@ -691,6 +691,11 @@
         "xmlbuilder2": "^3.0.2"
       }
     },
+    "node_modules/@cyclonedx/cyclonedx-library/node_modules/packageurl-js": {
```

Comment:
why is this? there should not be a duplicate install of  `packageurl-js`.
see https://github.com/CycloneDX/cyclonedx-javascript-library/issues/130